tests/test_.py

```
@@ -0,0 +1,232 @@
+        try:
+            data = json.loads(resp.content.decode()) if hasattr(resp, "content") else resp.json()
+            assert isinstance(data, (dict, list))
+        except (json.JSONDecodeError, AttributeError):
+            # Not JSON or different response type; acceptable depending on endpoint
+            pass
+
+    @pytest.mark.parametrize(("path", "payload", "expected_status"), [
+        ("/api/v1/resource/", {"name": "ok", "value": 1}, 201),
+        ("/api/v1/resource/", {"name": ""}, 400),
+    ])
+    def test_post_endpoints_validate_payloads(self, api_client: Any, path: str, payload: Any, expected_status: int):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_post_endpoints_validate_payloads`

Add return type annotation: `None`

(ANN201)